docker/production.yml

```diff
@@ -80,20 +80,24 @@ services:
 
   # status service
   status:
+    image: docker.cdot.systems/status
```

Comment:
All of our deployable images for staging and production use a tag of `:staging` or `:production`, see https://github.com/Seneca-CDOT/telescope/blob/master/.github/workflows/docker-build-and-push.yml#L121.  We need to pick the right tag for our deployment type with each of the `image` definitions in this file.

My suggestion is to do the following:

1. Add a new environment variable near the top of https://github.com/Seneca-CDOT/telescope/blob/master/config/env.staging#L1-L39 named `DOCKER_DEPLOY_TAG=staging`
2. Do something similar and add a new environment variable near the top of https://github.com/Seneca-CDOT/telescope/blob/master/config/env.production#L1-L39 named `DOCKER_DEPLOY_TAG=production` 
3.Add tag to each of the `image` lines you modified in this file that looks like this: image: docker.cdot.systems/image:${DOCKER_DEPLOY_TAG:-latest}.  This says to use the `DOCKER_DEPLOY_TAG` if present, or `latest` if missing.